lib/elastic_apm/middleware.rb

```diff
@@ -13,7 +13,7 @@ def initialize(app)
     # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
     def call(env)
       begin
-        if running? && !path_ignored?(env)
+        if running? && !path_ignored?(env) && !ElasticAPM.current_transaction
```

Comment:
@mikker I had to add this because otherwise the grape call to this method would try to create a transaction within a Rails transaction.

Reply:
I think this looks completely fine 👍